styleguide/screens/Patterns/Carousel/Carousel.js

```diff
@@ -0,0 +1,252 @@
+import React from 'react';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import Specimen from '../../../components/Specimen/Specimen';
+import { D, H, T, A, C } from '../../../components/Scaffold/Scaffold';
+
+import Carousel from '../../../../components/Carousel/Carousel';
+import PictureCard from '../../../../components/Cards/PictureCard/PictureCard';
+import ControlledCarousel from '../../../../components/Carousel/ControlledCarousel';
+
+import shared from '../../../../styleguide/shared.css';
+import m from '../../../../globals/modifiers.css';
+import css from './Carousel.css';
+
+const CarouselDocumentation = () => (
+  <div>
+    <H level={ 1 }>Carousel</H>
+    <T elm="p" className={ cx(m.mtr, m.largeI, m.demi) }>
+      Carousels help notify the user that the application is loading content.
```

Comment:
I think this might be the wrong paragraph 😅

Reply:
🤣